src/app/apolloClient.ts

```diff
@@ -9,6 +22,20 @@ const apolloClient = new ApolloClient({
     };
     operation.setContext({ headers });
   },
+  typeDefs,
+  resolvers: {
+    Query: {
+      async currentUser() {
+        const user = await authService.getUser();
+
+        if (!user) return null;
+
+        const { name, email, sub } = user.profile;
+
+        return { __typename: 'CurrentUser', id: sub, name, email };
+      },
+    },
+  },
```

Comment:
I wonder if we should store the user in Apollo store or Redux since we already have it in place.
For me, I don't have a strong opinion on either.

Reply:
I did this the way it's done in admin, so that both projects are familiar at least in this aspect. I also don't have any strong opinions one way or another 🤷  We can change this pretty easily if we come up with something else when we refine this feature